mst/postgresql-read-replica.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Create a read-only replica of PostgreSQL
+excerpt: Learn how to create a read-only replica for PostgreSQL service on Managed Service for TimescaleDB 
+product: mst
+---
+
+# Create a read-only replica of PostgreSQL service
+
+PostgreSQL® read-only replicas enable you to perform read-only queries against
+the replica and reduce the load on the primary server. You can optimise query
+response times across different geographical locations because the replica can
+be created in different regions or on different cloud providers.
+
+<highlight type="note">
+If your PostgreSQL service is running a [Pro plan](https://docs.timescale.com/mst/latest/about-mst/#service-configuration-plans),
+you have standby nodes available in a high availability setup. These support
+read-only queries to reduce the effect of slow queries on the primary node.
+</highlight>
+
+<procedure>
+
+## Creating a replica of POstgreSQL
+
+1.  Log in to the Aiven web console.
+
+1.  Select the PostgreSQL instance for which you want to create a remote replica.
+
+1.  In the `Overview` tab, click `Create a read replica` button.
+
+1.  In the `Create a PostgreSQL read` page, type a name for the remote replica,
```

Comment:
Is that the right name for the page?

Reply:
> Is that the right name for the page?

<img width="883" alt="Screenshot 2022-09-09 at 8 41 10 AM" src="https://user-images.githubusercontent.com/37059749/189264370-548885d6-893e-4a3c-9b67-f8647fa7e23a.png">